src/test/java/uk/gov/companieshouse/service/DissolutionGetterTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.companieshouse.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.fixtures.DissolutionFixtures;
+import uk.gov.companieshouse.mapper.DissolutionResponseMapper;
+import uk.gov.companieshouse.model.db.Dissolution;
+import uk.gov.companieshouse.model.dto.DissolutionGetResponse;
+import uk.gov.companieshouse.repository.DissolutionRepository;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class DissolutionGetterTest {
+
+    @InjectMocks
+    private DissolutionGetter getter;
+
+    @Mock
+    private DissolutionRepository repository;
+
+    @Mock
+    private DissolutionResponseMapper responseMapper;
+
+    @Test
+    public void get_findsDissolution_mapsToDissolutionResponse_returnsGetResponse() {
```

Comment:
What if nothing is returned? We should verify the mapper isn't called and Optional.empty is returned.